Shift3/boilerplate-client-react
# Test(Footer): Add unit tests

## Changes
1. Added footer.spec.tsx
2. Added Wrapper.spect.sx

## Purpose
To add unit tests to the footer component.

## Approach
There were no tests before, now they have been added.

## Testing
1. Pull in the changes to your local copy of this branch.
2. Run [`yarn test`]
3. Check that all tests have passed.

## Screenshots
![Screen Shot 2021-05-11 at 10 16 20 AM](https://user-images.githubusercontent.com/59614789/117830911-141d9200-b242-11eb-8fe6-bf499b41fd0f.png)
### Closes #124
